logging/source/EnvVariableParser.cpp

```diff
@@ -45,6 +46,16 @@ std::string getRialtoConsoleLog()
     return "";
 }
 
+std::string getRialtoLogPath()
+{
+    const char *debugVar = getenv("RIALTO_LOG_PATH");
```

Comment:
change variable name to logPath or something like that

Reply:
done